src/app/clusters/scenes-server/SceneHandlerImpl.cpp

```diff
@@ -20,6 +20,29 @@
 namespace chip {
 namespace scenes {
 
+namespace {
+
+/// ValideAttribute
+/// @brief  Validate the attribute exists for a given cluster
+/// @param[in] clusterID  Cluster ID
+/// @param[in] attID      Attribute ID
+/// @return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE if the attribute does not exist for a given cluster or is not scenable
+/// @note This will allways fail for global list attributes. If we do want to make them scenable someday, we will need to
+///       use a different validation method.
+// TODO: Assess if we also want to throw an error if the attribute value is out of range
```

Comment:
Possibly also if the type is incorrect.

Reply:
Yes, but that should be checked by the "S" quality since a scenable attribute must be one of the accepted types.

Reply:
The defined behavior in the spec is to cap the value at the maximal range accepted for the specific attribute in the event that a value greater than the expected one is stored. 

I added a check on the value, that way there will be a cap if a uint64 is passed to a uint8 attribute, however the check on maximal allowed value is more tedious as it is a per cluster / per attribute check.

Attributes in our data model do not currently have a "max" or "min" attribute that I know of

Reply:
Oh, interesting. so you can use a different type than the attribute and that's fine?

Reply:
At least from the Scene Table's perspective, and our TLV Encoder already reduce the save size depending on the value, so as long as it's an integer it is okay.